krabcake/rs_hello/src/lib.rs

```diff
@@ -823,23 +792,20 @@ pub extern "C" fn rs_shadow_qop(
         if_sb_event_queued_print(
             || msg!(b"rs_shadow_qop sb event \0"),
             || {
-                if PRINT_MSG { ppIROp(op); }
+                if PRINT_MSG {
```

Comment:
you missed restoring some of these

Reply:
even with Max heuristics, some of these were being broken up by `cargo fmt`, so I made a helper function instead.